SoftLayer/managers/image.py

```diff
@@ -68,7 +68,7 @@ def list_private_images(self, guid=None, name=None, **kwargs):
         kwargs['filter'] = _filter.to_dict()
 
         account = self.client['Account']
-        return account.getPrivateBlockDeviceTemplateGroups(**kwargs)
+        return account.getPrivateBlockDeviceTemplateGroups(**kwargs, limit=limit)
```

Comment:
Change this to `self.client.iter_call`

Reply:
Hi @caberos,

I have doubts.

I saw that you have added code **self.client.iter_call** in **def list_public_images()** function but **not** **added** in **def list_private_images()** function.

**Doubts:**
1. Is it possible to add self.client.iter_call in def list_private_images() function?
   a. If yes, can you please add it,
   b. If no, can you please share the reason behind it.?

2. I see that 
File path: softlayer-python/SoftLayer/fixtures/SoftLayer_Virtual_Guest_Block_Device_Template_Group.py

There are no fixtures of related to private images.

Thanks,
Ramkishor Chaladi.